api.ts

```diff
@@ -2203,3 +2296,49 @@ export function maybeRewriteConnectionForFormula<
   }
   return formula;
 }
+
+// This helper method finds any inline autocomplete functions in a static sync table schema.
+// These functions will need to be extracted into the "namedAutocompletes" property on the sync
+// table and replaced with strings.
+function listPropertiesWithAutocompleteFunctions(schema: ObjectSchemaDefinition<string, string>): string[] {
+  const result: string[] = [];
+  for (const propertyName of Object.keys(schema.properties)) {
```

Comment:
There can be nested properties, but presumably we don't need to recurse here because only top-level properties are editable and hence only those can actually execute their autocomplete functions? Probably worth an explanation in a comment.

Reply:
Correct, I've added a note:

```
// Not that we won't detect autocomplete functions within nested object schemas, but that's not necessary
// here: the only types you can autocomplete in a 2-way schema are the top-level ones.
```